### ISSUE TITLE
Reset / Auswahl für Voreinstellungen

### ISSUE THREAD

**Issue by cukabeka** [DonLoron/media_effect_manager#10]
Fände toll, wenn im Addon die Möglichkeit bestünde, bestimmte Voreinstellungen zu laden. Für Galerien etc habe ich etwa andere Anforderungen als für Teaser oder Profilbilder. Wenn ich einmal das Set geändert habe, ist es schwer, wieder was sinnvolles zusammenzustellen.

**Comment by DonLoron** [comment]
Verstehe ich richtig, dass du quasi mehrere Standard-Sets anlegen können möchtest?

**Comment by cukabeka** [comment]
Ja, genau, das meine ich. Praktisch wäre auch, wenn evtl schon ein paar Sets mitgeliefert sind, an denen ich mich orientieren kann.

Ein paar Szenarios gibt es ja fast in jedem Projekt zB

- Fullscreeen
- Teaser
